java/rocksjni/env.cc

```diff
@@ -77,3 +79,33 @@ void Java_org_rocksdb_RocksMemEnv_disposeInternal(
     JNIEnv* env, jobject jobj, jlong jhandle) {
   delete reinterpret_cast<rocksdb::Env*>(jhandle);
 }
+
+/*
+ * Class:     org_rocksdb_RocksHdfsEnv
+ * Method:    createHdfsEnv
+ * Signature: ()J
+ */
+jlong Java_org_rocksdb_RocksHdfsEnv_createHdfsEnv(
+    JNIEnv* env, jclass jclazz, jstring fsname) {
+  const char *fsNameNative = env->GetStringUTFChars(fsname, NULL);
```

Comment:
Please add some error checking here, `GetStringUTFChars` can return `nullptr` if an OutOfMemoryError occurs. Consider using `JniUtil::copyString` from here: https://github.com/adamretter/rocksdb/commit/41f4747dd665cdfc16f93f11207b625b7cd9a9cb#diff-e139fd8f10bb0f956a1d95ad73d1db06R939 and checking `has_exception`